src/edit_distance.jl

```diff
@@ -0,0 +1,116 @@
+"""
```

Comment:
can you insert the signature here, with an indentation, so that it appears in the help/docs?
something like 
```julia
   function edit_distance(G₁, G₂; insert_cost::Function=v->1.0,
                     delete_cost::Function=u->1.0,
                     subst_cost::Function=(u,v)->0.5,
                     heuristic::Function=DefaultEditHeuristic)
```